backend/services/prisonerProfileService.ts

```diff
@@ -65,6 +69,19 @@ export default ({
 
     const systemContext = await systemOauthClient.getClientCredentialsTokens(username)
 
+    // TODO: Temporary measure to allow 3rd party MegaNexus time to implement caching and other techniques to their API so it can handle high volume of calls. To be refactored!
+    const getNeurodivergenceSupportNeed = async (agencyId) => {
+      if (canViewNeurodivergenceSupportData(agencyId, neurodiversityEnabledPrisons as string)) {
+        const [divergenceData] = await Promise.all(
+          [curiousApi.getLearnerNeurodivergence(systemContext, offenderNo)].map((apiCall) =>
```

Comment:
As stated in the other PR - you ought to just use the standard promise in this situation.

But it isn't that important

Reply:
Point noted, will do in next PR